docs/uninstall.md

```diff
@@ -0,0 +1,21 @@
+# Uninstalling AzureAuth
+
+We provide an uninstall script you can use to Uninstall AzureAuth. The script removes the default AzureAuth reference in the PATH and it will deletes the AzureAuth installation folder.
+
+The script currently doesn't support uninstalling from custom locations. This is to avoid removing any potential files that are not safe to delete. If there are any potential installations in custom locations found in the PATH (installed through the `$AZUREAUTH_INSTALL_DIRECTORY` environment variable), a warning will be prompted showing its location. Installations in custom locations that are not listed in the PATH cannot be found and uninstalled.
+
+## Usage
+
+Run the following in PowerShell:
+
+```
+$script = "${env:TEMP}\uninstall.ps1"
+$url = "https://raw.githubusercontent.com/AzureAD/microsoft-authentication-cli/main/install/uninstall.ps1"
+[Net.ServicePointManager]::SecurityProtocol = [Net.SecurityProtocolType]::Tls12
+Invoke-WebRequest $url -OutFile $script; if ($?) { &$script }; if ($?) { rm $script }
+```

```

Comment:
Will this be added to AzureAuth `ReadMe` doc?

Reply:
In fact we should add it here - https://github.com/AzureAD/microsoft-authentication-cli/#installation

Reply:
I'd like to hear your opinion on that. I was thinking maybe the docs folder is enough

Reply:
IMO, the doc here is good and just a reference to this doc from `README` would be good too and that's what Shalini is suggesting, I think.

Reply:
Yes, what @mvanchaa said. I _think_ a reference is needed to ReadMe for discoverability.

Reply:
Personally, I dunno if I think it's necessary on the `README`. Usually people go to the `README` as a sort of "quickstart" guide. We could _link_ to the uninstall docs I guess, but I think I like having this as a separate topic in the `docs/` directory.

Reply:
Sorry for the confusion I may have caused! I did mean we keep it under `docs`. I am suggesting adding this as a reference to `README` as a one stop for discoverability.

Reply:
Nothing to be sorry about here. A reference to `README` or a reference in the `README` to `docs/uninstall.md`?

Reply:
The latter.

Reply:
Something like this in the Windows Install section?

`Uninstalling can be done with the uninstallation script. [Read more](docs/uninstall.md)`

Reply:
Yep, perfect!

Reply:
@shalinikhare27 @mvanchaa @reillysiemens I added it as another Note in the ReadMe. Let me know if it looks good to you :)